nebula/ui/components/VPNToggleCard.qml

```diff
@@ -72,6 +72,8 @@ Item {
             Layout.preferredWidth: 45
             checked: (!VPNSettings.protectSelectedApps)
             onClicked: handleClick()
+            accessibleName: label.text
+            Accessible.description: subLabel.text
```

Comment:
```suggestion
            Accessible.description: sublabel.text
```

also, IIRC `Accessible.description` will overwrite `Accessible.name` so I am not sure that this would give you the result you would expect (the combination of the two)